api/src/pcapi/core/educational/commands.py

```diff
@@ -68,7 +68,8 @@ def generate_fake_adage_token(readonly: bool) -> None:
     help="Overide previous ministry if needed.",
 )
 @click.option("--final", type=bool, is_flag=True, default=False, help="Flag deposits as final.")
-def import_deposit_csv(path: str, year: int, ministry: str, conflict: str, final: bool) -> None:
+@click.option("--not-dry", type=bool, is_flag=True, default=False, help="Commit the changes.")
```

Comment:
Je ne suis vraiment pas fun de la négation sur `--not-dry`.
Je pensais que tout le monde était aligné sur le `--dry-run` avec un défaut à True, mais je vois de tout dans le code et je suis triste.

Reply:
c'est vrai que sur les option Click on a seulement des --dry-run, je vais changer